spec/unit/rest_spec.rb

```diff
@@ -37,6 +37,32 @@
       expect_any_instance_of(Net::HTTP).to receive(:open_timeout=).with(5).and_call_original
       client.rest_api(:get, path)
     end
+
+    it 'supports following redirects' do
+      response1 = FakeResponse.new({ name: 'New' }, 301, location: path)
+      response2 = FakeResponse.new({ name: 'New' }, 200, location: path)
+      allow_any_instance_of(Net::HTTP).to receive(:request).and_return(response1)
+      allow_any_instance_of(Net::HTTP).to receive(:request).and_return(response2)
```

Comment:
OK, it's not the prettiest thing, but I got it to work and properly mock the 301 followed by a 200 response.